octoprint_mrbeam/software_update_information.py

```diff
@@ -479,35 +482,95 @@ def _get_curent_version(input_moduleconfig, module_id, plugin):
     return current_version
 
 
-def _generate_config_of_beamos(moduleconfig, beamos_date, tierversion):
+class VersionComperator:
+    """
+    Version Comperator class to compare two versions with the compare method
+    """
+
+    def __init__(self, identifier, priority, compare):
+        self.identifier = identifier
+        self.priority = priority
+        self.compare = compare
+
+    @staticmethod
+    def get_comperator(comparision_string, comparision_options):
+        """
+        returns the comperator of the given list of VersionComperator with the matchin identifier
+        Args:
+            comparision_string (str): identifier to search for
+            comparision_options (list:
+
+        Returns:
+
+        """
+        for item in comparision_options:
+            if item.identifier == comparision_string:
+                return item
+
+
+def _generate_config_of_beamos(moduleconfig, beamos_version, tierversion):
     """
-    generates the config for the given beamos_date of the tierversion
+    generates the config for the given beamos_version of the tierversion
+
     Args:
-        moduleconfig: update config of the module
-        beamos_date: date of the beamos
-        tierversion: software tier
+        moduleconfig (dict): update config of the module
+        beamos_version (str): version of the beamos
+        tierversion (str): software tier
 
     Returns:
         beamos config of the tierversion
```

Comment:
here, Please mention type then description of the return  ex. 
`(dict): beamos config of the tierversion`